src/nvim/option_defs.h

```diff
@@ -29,6 +29,7 @@
 #define P_RBUF         0x4000U      ///< redraw current buffer and recompute text
 #define P_RALL         0x6000U      ///< redraw all windows
 #define P_RCLR         0x7000U      ///< clear and redraw all
+#define P_RTAL         0x8000U      ///< redraw tabline
```

Comment:
This will cause tabline to be redrawn for every option with `P_COMMA`

Reply:
I think you can try some other value like `0x400U` or `0x800U`

Reply:
I don't know how but I completely missed P_COMMA 🤦🏽‍♂️. ok, let's go with 0x800u then. by the way, these are setting multiple bits can't this scheme make a flag be triggered by another flag if one is a subset of another?